code/modules/library/lib_items.dm

```diff
@@ -283,6 +284,68 @@
 	else
 		return ..()
 
+/obj/item/weapon/book/mindbook
+	name = "A Harmless Book"
+	icon = 'icons/obj/library.dmi'
+	icon_state ="book"
+	unique = 1
+	forbidden = 1
+	var/hallucinatory_cooldown
+
+/obj/item/weapon/book/mindbook/attack_self(mob/living/H)
+	if(ishuman(H))
+		var/mob/living/carbon/human/P = H
+		if(!(P.mind && P.mind.assigned_role == "Psychiatrist"))

```

Comment:
Yes, but if it's really psych only, `supply = 0` should be in the uplink datum, else you get it in the 20tc crate, and completely wasted, the other items you still can somewhat use normally.